client/capstone-app/src/component/DocInfo.js

```diff
@@ -60,26 +60,39 @@ export default function Doclist() {
   const [name, setName] = useState("");
   const [docData,setDocData]=useState(Docdata())
   const [num, setNum] = useState(0);
+  const[data,setdata] = useState([]);
   
   useEffect(()=>{
     localStorage.setItem("Num", num);
   },)
     
+  useEffect(()=>{
+    fetch("http://localhost:2917/User")
+        .then((response) => response.json())
+        .then((data) => {
+          // console.log(data);
+          setdata(data);
+        })
+        .catch((error) => {
+          console.log(" failed to fetch");
+        })
+  },[])
   
+ 
   return (
     <main className="DocContainer">
       {/* <div>
       {props.imagedata.map((abd)=>{
           return <img src={abd.img} alt=""/>
         })}
       </div> */}
-      {docData.map((abc) => {
+      {data.map((abc) => {
```

Comment:
Instead of writing "abc" as a variable name use more effective variable names. Refer (https://medium.com/@ConnorFinnegan/tips-for-writing-effective-variable-names-ed869a4cfa9e).